wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -357,7 +386,7 @@ def _stop_run(self, run_id: str) -> bool:
             )
             return False
 
-        if run.state == RunState.DEAD:
+        if not run.state.is_alive:
```

Comment:
Depending on the unknown state, if a run is in an unknown state for some time, should it be cleaned up?

Reply:
Yeah i have been struggling with this. There sort of is a ~time~ component here... We don't want to kill unknown state runs immediately, but we also don't want them to be in the scheduler forever... Any suggestions on how to deal with something like that? We could do (if state is unknown 2 times in a row) for slightly better handling, but it might be too aggressive. Hm.... @KyleGoyette

Reply:
Hard to say without knowing the specific cases where we'll see unknown. Are there cases where unknown states recover? Without knowing anything, a 5 minute wait on unknown runs (assuming that doesn't mean queued on the cluster) should work. But I'd like more answers on when a run is in an unknown state before okaying that plan